python-avd/pyavd/_eos_designs/structured_config/base/__init__.py

```diff
@@ -426,11 +422,8 @@ def spanning_tree(self) -> dict | None:
 
     @cached_property
     def service_unsupported_transceiver(self) -> dict | None:
-        """service_unsupported_transceiver based on unsupported_transceiver data-model. TODO: Add to schema - maybe as hidden."""
-        if (unsupported_transceiver := get(self._hostvars, "unsupported_transceiver")) is not None:
```

Comment:
so the old behavior looked at the variable `unsupported_transceiver` but the new is to look for `service_unsupported_tranceiver`. That would be a breaking change. Please rename the key in `eos_designs` to `unsupported_transceiver`.

Reply:
Fixed!